js/form-validity.js

```diff
@@ -9,6 +9,15 @@
   var timeIn = choiceForm.querySelector('select[name="timein"]');
   var timeOut = choiceForm.querySelector('select[name="timeout"]');
 
+
+  var disableAtMapActivation = function () {
```

Comment:
```suggestion
  var disableOnMapActivation = function () {
```
[Предлог по](https://translate.google.com/#view=home&op=translate&sl=en&tl=ru&text=on)